### DISCUSSION TITLE
[REQ] Allow the target_puzzle_hash to be specified for self pooling to an alternative/cold wallet

### DISCUSSION THREAD

**Discussion by daverof** [Chia-Network/chia-blockchain#9701] [Ideas]
When self-pooling, the manual `plotnft claim` of the 1.75 XCH can only be transferred to the current active address of the host wallet, which is of course a hot wallet.

Can you please provide a way to edit this `target_puzzle_hash` to specify an alternative address, to allow users to claim directly to cold wallets, to avoid having to manually transfer coins from the hot to the cold wallet every time a block is won?

https://github.com/Chia-Network/chia-blockchain/blob/13ff7489b606d38b8294ed8c256d0177d39eb4bb/chia/pools/pool_wallet_info.py#L68-L69

https://github.com/Chia-Network/chia-blockchain/blob/13ff7489b606d38b8294ed8c256d0177d39eb4bb/chia/rpc/wallet_rpc_api.py#L569

**Comment by daverof** [comment]
9 weeks later and I am seeing the same behavior in 1.3.
Can this please be looked at? It's probably only a couple of lines of code to fix.
Thanks

**Comment by RnMss** [comment]
Also, allow the delay_puzzle_hash to be specified before creating an plotnft.
